bedevere/bpo.py

```diff
@@ -40,16 +34,24 @@
     issue_number_found = ISSUE_RE.search(event.data["pull_request"]["title"])
     if not issue_number_found:
         issue = await util.issue_for_PR(gh, event.data["pull_request"])
-        status = SKIP_ISSUE_STATUS if util.skip("issue", issue) else FAILURE_STATUS
+        status = (SKIP_ISSUE_STATUS if util.skip("issue", issue)
+                                    else create_failure_status_no_issue())
     else:
-        if "body" in event.data["pull_request"]:
-            body = event.data["pull_request"]["body"] or ""
-            if not body or CLOSING_TAG not in body:
-                issue_number = issue_number_found.group("issue")
-                new_body = BODY.format(body=body, issue_number=issue_number)
-                body_data = {"body": new_body}
-                await gh.patch(event.data["pull_request"]["url"], data=body_data)
-        status = create_success_status(issue_number_found)
+        issue_number = issue_number_found.group("issue")
+        try:
+            issue_number_on_bpo = await _validate_issue_number(issue_number)
+        except NameError:
+            issue_number_on_bpo = True
+        if issue_number_on_bpo:
+            if "body" in event.data["pull_request"]:
+                body = event.data["pull_request"]["body"] or ""
+                if not body or CLOSING_TAG not in body:
+                    new_body = BODY.format(body=body, issue_number=issue_number)
+                    body_data = {"body": new_body, "maintainer_can_modify": True}
```

Comment:
Is `"maintainer_can_modify"` required for this to work, or are you just sneaking that in here? If it's the latter then I would pull it out as it's easy to lose track of. Better to open an issue and do a separate check for it as this isn't related to issue numbers from bugs.python.org which is what this module is aimed at.